make/lib/Awt2dLibraries.gmk

```diff
@@ -0,0 +1,1130 @@
+#
```

Comment:
looks like this file duplicates the jdk/make/lib/Awt2dLibraries.gmk?

Reply:
correct, I'll remove it